src/components/Selection.spec.js

```diff
@@ -5,7 +5,7 @@ import Selection from './Selection';
 describe('Selection', () => {
```

Comment:
There are tests missing with this component. You should always test the props you are passing.